source/use-cases/sensitive-data-encryption.txt

```diff
@@ -75,20 +75,106 @@ velit congue vel.
 D. Perform Encrypted Read/Write Operations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Etiam hendrerit posuere justo, ac hendrerit lorem pellentesque sed.
-Fusce ullamcorper neque eu libero aliquam, sed placerat sapien cursus.
-Vestibulum maximus magna sed velit feugiat tempor.
-
-Ut accumsan pellentesque metus, at porta felis dapibus vulputate. Aenean
-malesuada interdum libero in feugiat. Fusce pharetra massa eget leo
-varius, et gravida massa tempus.
-
-Ut mattis eros elit, sed ultrices purus lacinia sed. Curabitur tincidunt
-tellus eget neque porta dictum ac vel orci. Vestibulum ante ipsum primis
-in faucibus orci luctus et ultrices posuere cubilia Curae.
-
-Praesent feugiat arcu non risus elementum eleifend. Nullam sed iaculis
-erat.
+Read and write operations on encrypted fields use the same methods and have the capabilities of traditional MongoDB read and write operations with the added security of field-level encryption.
+
+MedcoMD engineers continue the development of the Medical Care Management 
+System. In the system, doctors should be able to read and write to all fields, 
+and receptionists should only be allowed to read and write to non-sensitive fields. 
+
+MedcoMD engineers create a function to insert a new patient:
+
+
+.. tabs::
+
+   tabs:
+
+     - id: java-jsonschema-generation
+       name: "Java"
+       content: |
+       
+        .. code-block:: java
+
+            public static void insertPatient(MongoCollection collection, String name, int ssn, String bloodType, ArrayList<Document> medicalRecords, int policyNumber, String provider) {
+                Document patient = new Document();
+                patient.append("name", name);
+                patient.append("ssn", ssn);
+                patient.append("bloodType", bloodType);
+                patient.append("medicalRecords", medicalRecords);
+                Document insurance = new Document();
+                insurance.append("policyNumber", policyNumber);
+                insurance.append("provider", provider);
+                patient.append("insurance", insurance);
+                collection.insertOne(patient);
+            }
+
+
+When a new record is inserted using a client configured to use CSFLE,
+such as when a doctor inserts a new patient in the MedcoMD Management
+System, the sensitive fields become encrypted:
```

Comment:
I could be mistaken, but I think "become" implies "will become" which is passive voice. Instead, you could use "are".